Messenger/AppDelegate.mm

```diff
@@ -120,7 +121,7 @@ - (void)applicationDidFinishLaunching:(NSNotification*)notification {
   if ([[NSUserDefaults standardUserDefaults] boolForKey:@"moves-with-active-space"]) {
     _window.collectionBehavior |= NSWindowCollectionBehaviorMoveToActiveSpace;
   }
-  _window.minSize = {605,300};
+  _window.minSize = {400,300};
```

Comment:
Have you tested this? Last time I tried this (as the comment below suggests) the view started scrolling horizontally.